src/Visualization/SolarSystemPane.java

```diff
@@ -77,39 +102,38 @@ public void addGrid()
 
         //Draw grid lines
         unitSize = 50;
-        for( double i=unitSize; i < width; i+=unitSize)
-        {
-            gc.strokeLine( i, 0, i, height); //Vertical lines = 40
-            gc.strokeLine( 0, i, width, i);  //Horizontal lines = 46
+        for( double i=unitSize; i < width*2; i+=unitSize) {
+            gc.strokeLine( i, 0, i, height*2); //Vertical lines = 40
+            gc.strokeLine( 0, i, width*2, i);  //Horizontal lines = 46
         }
 
-        space.getChildren().add(grid);
+        solarSystem.getChildren().add(grid);
     }
 
     public Canvas getGrid() { return grid; }
 
     /**
-     * Method to convert km into AU
+     * Method to convert km into AU - change into m to AU
      */
-    public double fromKmToAU(double km)
-    {
-        return km/1.496e+8;
+    public double fromMToAU(double m) {
+        double AU = m*6.6846e-12;
+        return AU;
     }
 
+    //Add planets to solar system
+    //1 unit of the grid = 50 pixels = 1 AU
+    //Coordinates of the planets = 50 * planet's distance from the sun (AU)
     /**
-     * Add 'planet name' to the solar system
+     * Add 'planet name'
      * Add -/+ depending on the quadrant of the cartesian plane the planet is in
-     * Coordinates of the planets = 50 (1 unit of the grid = 50 pixels = 1 AU) * planet's distance from the sun (AU)
-     * The planets are initialized with their coordinates on 01-04-2020
      * @param x x-coordinate of the planet = AU from sun
      * @param y y-coordinate of the planet = AU from sun
      */
-   public void addSun(double x, double y)
-   {
+   public void addSun(double x, double y) {
        double xSun = xCenter + x;
        double ySun = yCenter + y;
        sun = new Circle(xSun, ySun, 10);
-       Image imageSun = new Image(getClass().getResourceAsStream("/Visualization/Resources/Sun.png"));
+       Image imageSun = new Image(getClass().getResourceAsStream("Resources/Sun.png"));
```

Comment:
I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [UI: Usage of GetResource may be unsafe if class is extended (UI_INHERITANCE_UNSAFE_GETRESOURCE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#ui-usage-of-getresource-may-be-unsafe-if-class-is-extended-ui-inheritance-unsafe-getresource).
Calling this.getClass().getResource(...) could give results other than expected if this class is extended by a class in another package.